Github client panic v1.3.0

Hey guys

Just pulled v1.3.0 (new app capability, yay!) - hitting this during pod startup. 

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x29a4a69]

goroutine 1 [running]:
github.com/zapier/kubechecks/pkg/vcs/github_client.CreateGithubClient()
	/src/pkg/vcs/github_client/client.go:76 +0x409
github.com/zapier/kubechecks/cmd.createVCSClient({0xc000058074?, 0x334550f?})
	/src/cmd/vcs.go:16 +0x45
github.com/zapier/kubechecks/cmd.glob..func1(0x58a4ae0?, {0x333c6b4?, 0x0?, 0x0?})
	/src/cmd/controller_cmd.go:30 +0x4a
github.com/spf13/cobra.(*Command).execute(0x58a4ae0, {0x593e390, 0x0, 0x0})
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x58a50a0)
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/zapier/kubechecks/cmd.Execute()
	/src/cmd/root.go:37 +0xc5
main.main()
	/src/main.go:8 +0x17
```

sorry for being so annoying :-)

--- a/pkg/vcs/client.go
+++ b/pkg/vcs/client.go
@@ -9,6 +9,11 @@ import (
 	"github.com/zapier/kubechecks/pkg/repo"
 )
 
+const (
+	DefaultVcsUsername = "kubechecks"
+	DefaultVcsEmail    = "kubechecks@zapier.com"
+)
+
 var (
 	// ErrInvalidType is a sentinel error for use in client implementations
 	ErrInvalidType  = errors.New("invalid event type")

--- a/pkg/vcs/github_client/client.go
+++ b/pkg/vcs/github_client/client.go
@@ -69,12 +69,26 @@ func CreateGithubClient() (*Client, error) {
 		return nil, errors.Wrap(err, "failed to get user")
 	}
 
-	return &Client{
+	client := &Client{
 		Client:   googleClient,
 		v4Client: shurcoolClient,
-		username: *user.Login,
-		email:    *user.Email,
-	}, nil
+	}
+	if user != nil {
+		if user.Login != nil {
+			client.username = *user.Login
+		}
+		if user.Email != nil {
+			client.email = *user.Email
+		}
+	}
+
+	if client.username == "" {
+		client.username = vcs.DefaultVcsUsername
+	}
+	if client.email == "" {
+		client.email = vcs.DefaultVcsEmail
+	}
+	return client, nil
 }
 
 func (c *Client) Username() string { return c.username }

--- a/pkg/vcs/gitlab_client/client.go
+++ b/pkg/vcs/gitlab_client/client.go
@@ -55,7 +55,14 @@ func CreateGitlabClient() (*Client, error) {
 		return nil, errors.Wrap(err, "failed to get current user")
 	}
 
-	return &Client{Client: c, username: user.Username, email: user.Email}, nil
+	client := &Client{Client: c, username: user.Username, email: user.Email}
+	if client.username == "" {
+		client.username = vcs.DefaultVcsUsername
+	}
+	if client.email == "" {
+		client.email = vcs.DefaultVcsEmail
+	}
+	return client, nil
 }
 
 func (c *Client) Email() string    { return c.email }

